docs/tutorial/tutorial_05.rst

```diff
@@ -0,0 +1,169 @@
+Part 5 - Using Celery to Automate Maintenance Chores
+====================================================
+
+Scenario
+--------
+In :doc:`Part 1 <tutorial_01>` you created your own :term:`Authorization Server` and it's running along just fine.
+However, the database is getting cluttered with expired tokens. You can periodically run
+the :doc:`cleartokens management command <../management_commands>`, but why not automate this with
+`Celery <https://docs.celeryq.dev/>`_?
+
+Set up RabbitMQ
+---------------
+Celery components communicate via a message queue. We'll use `RabbitMQ <https://www.rabbitmq.com/>`_.
+
+Install Rabbit MQ on MacOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are using MacOS it's likely you are already using `Homebrew <https://brew.sh/>`_. If not, now's
+the time to start.
+
+::
+
+   brew install rabbitmq
+   brew service start rabbitmq
+
+Install RabbitMQ with Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This will start up a docker image that just works:
+::
+
+   docker run -it --rm --name rabbitmq -p 5672:5672 -p 15672:15672 rabbitmq:3.9-management
+
+
+
+Install RabbitMQ on Windows
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+See the `RabbitMQ Installing on Windows <https://www.rabbitmq.com/install-windows.html>`_ instructions.
+
+
+Add Celery
+----------
+Make sure you virtualenv is active and install `celery` and
+`django-celery-beat <https://django-celery-beat.readthedocs.io/>`_.
+
+::
+
+    pip install celery django-celery-beat
+
+Update your list of installed apps to include both your AS server app -- we'll call it ``tutorial``,
+and ``django_celery_beat`` which extends your Django project to store your periodic task schedule
+in the database and adds a Django Admin interface for configuring them.
+
+.. code-block:: python
+
+    INSTALLED_APPS = {
+        # ...
+        "tutorial",
+        "django_celery_beat",
+    }
+
+
+Now add a new file to your app to add Celery: ``tutorial/celery.py``:
+
+.. code-block:: python
+
+    import os
+
+    from celery import Celery
+
+    # Set the default Django settings module for the 'celery' program.
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'tutorial.settings')
+    app = Celery('tutorial', broker="pyamqp://guest@localhost//")
+    app.config_from_object('django.conf:settings', namespace='CELERY')
+
+    # Load task modules from all registered Django apps.
+    app.autodiscover_tasks()
+
+This will autodiscover any ``tasks.py`` files in the list of installed apps.
+We'll add ours now in ``tutorial/tasks.py``:
+
+.. code-block:: python
+
+    from celery import shared_task
+
+    @shared_task
+    def clear_tokens():
+	from oauth2_provider.models import clear_expired
+
+	clear_expired()
```

Comment:
```suggestion
    def clear_tokens():
		from oauth2_provider.models import clear_expired

		clear_expired()
```

Reply:
This is how it shows for me:

<img width="737" alt="Screenshot 2022-03-21 at 17 41 19" src="https://user-images.githubusercontent.com/96283/159297295-2d65ccd4-bff9-4118-a6db-6299b05f12de.png">

I thought the indentation was broken. If it's not, please ignore my comment. Maybe it's just a GitHub rendering issue.

Reply:
I see the file [shows up just fine](https://github.com/jazzband/django-oauth-toolkit/blob/fde14c0566606e4cf4c20ad73981549f0594a39e/docs/tutorial/tutorial_05.rst), so please ignore my comment.

Reply:
@alexei go figure, github doesn't render RST properly;-)

If you are able to approve the PR then I can merge it. I'm not sure whether only Jazzband members can be approvers though -- github wouldn't let me request you as a reviewer....

Reply:
Yeah, that's alright, it serves mostly those who are not aware of it. We've already upgraded to 1.7.1 and addressed the issue internally 👍